src/api-console-styles.html

```diff
@@ -267,45 +267,48 @@
     @apply(--arc-font-body1);
   };
 
+  /* buttons */
+
   --code-anypoint-robustblue2: #6b8a99;
   --code-anypoint-robustblue2-hover: #7d9eae;
-
   --code-anypoint-default-button-hover: #04b3f5;
 
-  --paper-toggle-button: {
-    @apply(--arc-font-body1);
-    border: 1px solid var(--code-anypoint-robustblue2);
-    color: var(--code-anypoint-robustblue2);
+  --paper-button: {
+    @apply(--anypoint-font-body);
+    font-size: 14px;
+    border-radius: 0;
+    box-shadow: none;
+    padding: 10px 16px;
+    box-sizing: border-box;
+    text-transform: none;
   };
 
-  --paper-toggle-button-hover: {
-    background-color: var(--code-anypoint-robustblue2-hover);
-    border-color: var(--code-anypoint-robustblue2-hover);
-    color: white;
-  };
+  --paper-button-ink-color: transparent;
 
-  --paper-primary-button: {
+  --action-button {
     @apply(--arc-font-body1);
     border: 1px solid var(--primary-color);
     color: white;
-  };
+  }
 
-  --paper-primary-button-hover: {
-    background-color: var(--code-anypoint-default-button-hover);
-    border-color: var(--code-anypoint-default-button-hover);
-    color: white;
-  };
+  --action-button-hover: {
+    background: var(--code-anypoint-default-button-hover);
```

Comment:
Why do we use this variable name? In the theme any `--code` names are reserver for code blocks (pre, code, code-mirror, etc). It's not a big deal but it can be misleading.